src/style.css

```
@@ -1 +1,75 @@
-/* Styles go here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  box-sizing: content-box;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+}
+
+.container {
+  padding: 0 50px;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.navigation-list {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+}
+
+.navigation li {
+  height: 100%;
+}
```

Дай класс элементу ли, и стилизуй его напрямую, а не через navigation
```suggestion
.liClass {
  height: 100%;
}
```